packages/manager/modules/pci/package.json

```diff
@@ -27,6 +27,7 @@
     "xterm": "^3.14.5"
   },
   "peerDependencies": {
+    "@adyen/adyen-web": "^3.20.0",
```

Comment:
Any reason to prefer a peer to a dependency ? (as it's a non angularjs dependency)

Reply:
Indeed it can be a `dependency`

Reply:
Dependency was set directly in `ng-ovh-payment-method` (it was declared as `peerDep` before in that module).